include/oneapi/tbb/parallel_scan.h

```diff
@@ -537,6 +538,40 @@ class lambda_scan_body {
     }
 };
 
+
+  template<typename BasePartitioner>
+  template<typename Range, typename Body>
+  void numa_partitioner<BasePartitioner>::execute_scan(const Range& range, Body& body) const{
```

Comment:
Let's skip parallel_scan completely. Already there are only a limited set of partitioners that work with parallel_scan. Let's not worry about making this work.